corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -327,6 +327,126 @@ hqDefine("geospatial/js/case_grouping_map",[
         );
     }
 
+    function caseGroupSelectModel() {
+        'use strict';
+        var self = {};
+
+        self.allGroups = ko.observableArray([]);
+        self.allCaseGroups;
+        self.visibleGroupIDs = ko.observableArray([]);
+        self.groupMaxSizeBreached = ko.observable(false);
+        self.casePerGroup = {};
+
+        self.groupIDInVisibleGroupIds = function(groupID) {
+            if (self.visibleGroupIDs().indexOf(groupID) !== -1) {
```

Comment:
Nit: This function could be shortened to a single line as `return self.visibleGroupIDs().indexOf(groupId) !== -1`.

Reply:
Lol thanks

Reply:
https://github.com/dimagi/commcare-hq/pull/33680/commits/b514f9e1390c71bc57c0e914b872458f0532afa6

Reply:
Thanks!